projects/packages/my-jetpack/_inc/components/product-card/style.module.scss

```diff
@@ -0,0 +1,98 @@
+:root {
+	--font-title-large: 36px;
+	--font-title-small: 24px;
+	--font-body: 16px;
+	--font-label: 12px;
+
+	--jp-black:           #000000;
+	--jp-black-80:        #2C3338;
+	--jp-white:           #FFFFFF;
+	--jp-white-off:       #F9F9F6;
+	--jp-gray:            #DCDCDE;
+	--jp-gray-off:        #E2E2DF;
+
+	--jp-green-primary:   #069E08;
+	--jp-green-secondary: #2FB41F;
+
+  --jp-red-0:           #F7EBEC;
+
+	--jp-border-radius:       4px;
+	--jp-menu-border-height:  1px;
+	--jp-underline-thickness: 2px;
+
+  --is-active:          #008710;
+  --is-inactive:        #646970;
+  --is-error:           #B32D2E;
+  --is-disabled:        #C3C4C7;
+}
```

Comment:
should we move this to a global file?

Reply:
actually, I was going to suggest using @import to keep-in-sync with Jetpack styles but not sure whether it's supported by CSS modules.

Reply:
@retrofox this will be removed in the next commits

Reply:
Done in a3fa81fafad91890725d1f360b25a3404c8a2cd0